internal/graphicsdriver/opengl/gl/package_darwin.go

```diff
@@ -0,0 +1,686 @@
+	gpClear                       uintptr
+	gpColorMask                   uintptr
+	gpCompileShader               uintptr
+	gpCreateProgram               uintptr
+	gpCreateShader                uintptr
+	gpDeleteBuffers               uintptr
+	gpDeleteFramebuffersEXT       uintptr
+	gpDeleteProgram               uintptr
+	gpDeleteRenderbuffersEXT      uintptr
+	gpDeleteShader                uintptr
+	gpDeleteTextures              uintptr
+	gpDisable                     uintptr
+	gpDisableVertexAttribArray    uintptr
+	gpDrawElements                uintptr
+	gpEnable                      uintptr
+	gpEnableVertexAttribArray     uintptr
+	gpFlush                       uintptr
+	gpFramebufferRenderbufferEXT  uintptr
+	gpFramebufferTexture2DEXT     uintptr
+	gpGenBuffers                  uintptr
+	gpGenFramebuffersEXT          uintptr
+	gpGenRenderbuffersEXT         uintptr
+	gpGenTextures                 uintptr
+	gpGetBufferSubData            uintptr
+	gpGetDoublei_v                uintptr
+	gpGetDoublei_vEXT             uintptr
+	gpGetError                    uintptr
+	gpGetFloati_v                 uintptr
+	gpGetFloati_vEXT              uintptr
+	gpGetIntegeri_v               uintptr
+	gpGetIntegerui64i_vNV         uintptr
+	gpGetIntegerv                 uintptr
+	gpGetPointeri_vEXT            uintptr
+	gpGetProgramInfoLog           uintptr
+	gpGetProgramiv                uintptr
+	gpGetShaderInfoLog            uintptr
+	gpGetShaderiv                 uintptr
+	gpGetTransformFeedbacki64_v   uintptr
+	gpGetTransformFeedbacki_v     uintptr
+	gpGetUniformLocation          uintptr
+	gpGetUnsignedBytei_vEXT       uintptr
+	gpGetVertexArrayIntegeri_vEXT uintptr
+	gpGetVertexArrayPointeri_vEXT uintptr
+	gpIsFramebufferEXT            uintptr
+	gpIsProgram                   uintptr
+	gpIsRenderbufferEXT           uintptr
+	gpIsTexture                   uintptr
+	gpLinkProgram                 uintptr
+	gpPixelStorei                 uintptr
+	gpReadPixels                  uintptr
+	gpRenderbufferStorageEXT      uintptr
+	gpScissor                     uintptr
+	gpShaderSource                uintptr
+	gpStencilFunc                 uintptr
+	gpStencilOp                   uintptr
+	gpTexImage2D                  uintptr
+	gpTexParameteri               uintptr
+	gpTexSubImage2D               uintptr
+	gpUniform1f                   uintptr
+	gpUniform1i                   uintptr
+	gpUniform1fv                  uintptr
+	gpUniform2fv                  uintptr
+	gpUniform3fv                  uintptr
+	gpUniform4fv                  uintptr
+	gpUniformMatrix2fv            uintptr
+	gpUniformMatrix3fv            uintptr
+	gpUniformMatrix4fv            uintptr
+	gpUseProgram                  uintptr
+	gpVertexAttribPointer         uintptr
+	gpViewport                    uintptr
+)
+
+func boolToInt(b bool) int {
+	if b {
+		return 1
+	}
+	return 0
+}
+
+func ActiveTexture(texture uint32) {
+	purego.SyscallN(gpActiveTexture, uintptr(texture))
```

Comment:
We might be integrate `package_windows.go` and `package_darwin.go` by adding a Windows version of purego.SyscallN. (Note that syscall.SyscallN is not available for Ebiten due to the current Go version restriction). Of course you don't have to do this in this PR.

Reply:
I think that would be a good idea. It should be simple enough: just separate out `SyscallN` into its own file and then implement `syscall_Sycall9` for windows using the `windows` package version